tests/runFromSimTest.m

```diff
@@ -119,31 +99,26 @@ function removeInputFiles(testCase)
     methods(Test)
         function fromSimCustom(testCase)
             % Run WEC-Sim from Simulink with custom parameters
-            cd(testCase.runFromSimDir)
-            
+            cd(testCase.runFromSimDir)            
             simFile = fullfile(testCase.runFromSimDir,'fromSimCustom.slx');
             load_system(simFile);
             run('initializeWecSim');
-            sim(simFile, [], simset('SrcWorkspace','current'));
-            
+            sim(simFile, [], simset('SrcWorkspace','current'));            
             close_system(simFile,0);
             bdclose('all')
             clear body constraint output pto simu waves
         end
         
         function fromSimInput(testCase)
             % Run WEC-Sim from Simulink with input file
```

Comment:
same with the empty lines here, I think its much harder to read and pick out sections of the code